pkgs/misc/vim-plugins/overrides.nix

```
@@ -764,4 +765,30 @@ self: super: {
         ${vim}/bin/vim --cmd ":set rtp^=$PWD" -c 'ru plugin/unicode.vim' -c 'UnicodeCache' -c ':echohl Normal' -c ':q' > /dev/null
       '';
   });
+
+  vim-clap = super.vim-clap.overrideAttrs(old: {
+    preFixup = let
+      maple-bin = rustPlatform.buildRustPackage {
+        name = "maple";
+        src = old.src;
+
+        nativeBuildInputs = [
+          pkgconfig
+        ];
+
+        buildInputs = [
+          openssl
+        ] ++ stdenv.lib.optionals stdenv.isDarwin [
+          CoreServices
+          curl
+          libgit2
+          libiconv
+        ];
+
+        cargoSha256 = "1rldd3rg8r2bczlv8iqws40bmf0k7is6nvi5qyg1mqqd3l6sml8i";
+      };
+    in ''
+      cp ${maple-bin}/bin/maple $target/bin/maple
```

can we just do a link, instead of copying?
```suggestion
      ln -s ${maple-bin}/bin/maple $target/bin/maple
```